[2023-10-15T15:11:17.729+0100] {processor.py:153} INFO - Started process (PID=29408) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:11:17.746+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:11:17.746+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:11:17.746+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:11:17.752+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:11:17.751+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:11:17.763+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:11:17.938+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.210 seconds
[2023-10-15T15:12:22.303+0100] {processor.py:153} INFO - Started process (PID=29531) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:12:22.306+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:12:22.306+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:12:22.306+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:12:22.312+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:12:22.311+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:12:22.312+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:12:22.561+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.258 seconds
[2023-10-15T15:13:33.051+0100] {processor.py:153} INFO - Started process (PID=29999) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:13:33.053+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:13:33.053+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:13:33.053+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:13:33.059+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:13:33.058+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:13:33.059+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:13:33.215+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.164 seconds
[2023-10-15T15:14:37.493+0100] {processor.py:153} INFO - Started process (PID=30219) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:14:37.496+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:14:37.496+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:14:37.496+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:14:37.506+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:14:37.505+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:14:37.506+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:14:37.669+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.177 seconds
[2023-10-15T15:15:39.580+0100] {processor.py:153} INFO - Started process (PID=30351) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:15:39.582+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:15:39.582+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:15:39.582+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:15:39.587+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:15:39.587+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:15:39.588+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:15:39.739+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.160 seconds
[2023-10-15T15:16:25.707+0100] {processor.py:153} INFO - Started process (PID=30449) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:16:25.709+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:16:25.710+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:16:25.710+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:16:25.716+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:16:25.715+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:16:25.716+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:16:25.884+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.177 seconds
[2023-10-15T15:19:57.319+0100] {processor.py:153} INFO - Started process (PID=30820) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:19:57.323+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:19:57.324+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:19:57.324+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:19:57.329+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:19:57.328+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:19:57.329+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:19:57.512+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.193 seconds
[2023-10-15T15:21:08.748+0100] {processor.py:153} INFO - Started process (PID=31009) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:21:08.750+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:21:08.751+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:21:08.751+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:21:08.756+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:21:08.755+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:21:08.756+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:21:08.929+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.181 seconds
[2023-10-15T15:22:11.150+0100] {processor.py:153} INFO - Started process (PID=31192) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:22:11.153+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:22:11.154+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:22:11.154+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:22:11.178+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:22:11.168+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:22:11.179+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:22:11.423+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.274 seconds
[2023-10-15T15:23:16.479+0100] {processor.py:153} INFO - Started process (PID=31348) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:23:16.481+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:23:16.482+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:23:16.482+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:23:16.487+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:23:16.486+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:23:16.487+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:23:16.654+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.174 seconds
[2023-10-15T15:24:21.425+0100] {processor.py:153} INFO - Started process (PID=31485) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:24:21.428+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:24:21.428+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:24:21.428+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:24:21.433+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:24:21.432+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:24:21.433+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:24:21.593+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.169 seconds
[2023-10-15T15:25:26.579+0100] {processor.py:153} INFO - Started process (PID=31611) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:25:26.581+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:25:26.581+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:25:26.581+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:25:26.586+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:25:26.585+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:25:26.586+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:25:26.758+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.180 seconds
[2023-10-15T15:26:31.609+0100] {processor.py:153} INFO - Started process (PID=31735) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:26:31.612+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:26:31.612+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:26:31.612+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:26:31.617+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:26:31.616+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:26:31.617+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:26:31.773+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.164 seconds
[2023-10-15T15:27:35.871+0100] {processor.py:153} INFO - Started process (PID=31843) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:27:35.874+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:27:35.874+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:27:35.874+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:27:35.879+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:27:35.878+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:27:35.879+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:27:36.159+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.288 seconds
[2023-10-15T15:28:40.577+0100] {processor.py:153} INFO - Started process (PID=31955) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:28:40.580+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:28:40.580+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:28:40.580+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:28:40.585+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:28:40.584+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:28:40.585+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:28:40.748+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.171 seconds
[2023-10-15T15:29:45.914+0100] {processor.py:153} INFO - Started process (PID=32076) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:29:45.917+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:29:45.917+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:29:45.917+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:29:45.922+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:29:45.921+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:29:45.922+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:29:46.077+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.163 seconds
[2023-10-15T15:30:49.823+0100] {processor.py:153} INFO - Started process (PID=32200) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:30:49.826+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:30:49.826+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:30:49.826+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:30:49.831+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:30:49.830+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:30:49.831+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:30:49.996+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.173 seconds
[2023-10-15T15:32:00.033+0100] {processor.py:153} INFO - Started process (PID=32329) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:32:00.037+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:32:00.038+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:32:00.038+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:32:00.043+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:32:00.042+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:32:00.043+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:32:00.209+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.176 seconds
[2023-10-15T15:33:03.016+0100] {processor.py:153} INFO - Started process (PID=32463) to work on /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:33:03.019+0100] {processor.py:743} INFO - Processing file /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2023-10-15T15:33:03.019+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:33:03.019+0100] {dagbag.py:532} INFO - Filling up the DagBag from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:33:03.024+0100] {logging_mixin.py:137} INFO - [2023-10-15T15:33:03.024+0100] {dagbag.py:341} ERROR - Failed to import: /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2023-10-15T15:33:03.025+0100] {processor.py:755} WARNING - No viable dags retrieved from /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py
[2023-10-15T15:33:03.223+0100] {processor.py:175} INFO - Processing /Users/b/Documents/workflow/venv/lib/python3.11/site-packages/airflow/example_dags/plugins/workday.py took 0.207 seconds
